packages/grafana-data/src/datetime/durationutil.test.ts

```diff
@@ -18,12 +18,12 @@ describe('Duration util', () => {
   describe('parseDuration', () => {
     it('parses a duration string', () => {
       const durationString = '3M 5d 20m';
-      expect(parseDuration(durationString)).toEqual({ months: '3', days: '5', minutes: '20' });
+      expect(parseDuration(durationString)).toEqual({ months: 3, days: 5, minutes: 20 });
```

Comment:
Could this cause issues for anyone using this util and expecting it to be a string?

Reply:
Yeah, this is a tricky one.

Currently, the function has a bug Its type indicates it returns numbers, but it actually returns strings. So no one _should_ be expecting strings. However, it does return strings. So 🤷🏻 

I'm happy to punt this to 9.0 if we think that's safer?

Reply:
Cool, lets merge this since it is a bug fix.